tools/gorm/print_schema_changes.sh

```diff
@@ -10,9 +10,32 @@
 
 git_branch=$1
 db_conn_string=$2
+# TODO: Remove when buildbuddy-internal#2096 is fixed
```

Comment:
I simplified the script in https://github.com/buildbuddy-io/buildbuddy/pull/3508, so rebased this clickhouse change off those simplications.